lib/metadata/metadataUtils.js

```diff
@@ -8,6 +8,33 @@ import { isBucketAuthorized,
     isObjAuthorized } from '../api/apiUtils/authorization/aclChecks';
 import bucketShield from '../api/apiUtils/bucket/bucketShield';
 
+exports.parseListEntries = entries =>
+    entries.map(entry => {
+        const tmp = JSON.parse(entry.value);
```

Comment:
I guess it's better to be prudent. I added another commit, can you check and see if it's OK?